scripts/bwc/utils.sh

```diff
@@ -67,3 +67,14 @@ function upload_data() {
   rm "$1.tar.gz"
   echo "Data has been uploaded and ready to test"
 }
+
+function get_dashboards_package_version() {
+  DASHBOARDS_PACKAGE_VERSION=$(cat $DASHBOARDS_DIR/package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g' \
+  | tr -d [:space:])
+  
+  echo "$DASHBOARDS_PACKAGE_VERSION"
+}
```

Comment:
This already exists (except for the directory specification) here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/f9e572d50be9bd7e9baf8b4387af40b0252cec4a/dev-tools/get-version.sh#L8-L13

Is there a reason we can't combine or re-use?

Reply:
Re-capping in offline conversation, 

The `bwc/utils.sh` gets used on release builds that are passed with the command and not the repo. The `dev-tools/get-version.sh` allows us to get the version from the git repo's `package.json` whereas the BWC version gets the version from the release build's `package.json`. Unfortunately dev-tools/get-version.sh does not get bundled with the release so that is the duplication here. There is an argument that can be made if we need to bundle it.